packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -110,6 +112,27 @@ export default class LocusInfo extends EventsScope {
     // return value ignored on purpose
     meeting.meetingRequest
       .getLocusDTO({url})
+      .catch((e) => {
```

Comment:
question: this is good - this will handle the 409 but isn't it possible that this would still get called in an infinite loop via

```
  applyLocusDeltaData(action: string, locus: any, meeting: any) {
    const {DESYNC, USE_CURRENT, USE_INCOMING, WAIT} = LocusDeltaParser.loci;

    switch (action) {
      case USE_INCOMING:
        meeting.locusInfo.onDeltaLocus(locus);
        break;
      case USE_CURRENT:
      case WAIT:
        // do nothing
        break;
      case DESYNC:
        this.doLocusSync(meeting);
        break;
      default:
        LoggerProxy.logger.info(
          `Locus-info:index#applyLocusDeltaData --> Unknown locus delta action: ${action}`
        );
    }
  }
  ```
  
  Shouldn't we *also* have a check for the meeting to be over by handling
  
  `2409113=Invalid client sequence data on state synchronization (check if meeting has ended).`
  
  i.e., if the sync request returns with 2409113 code we could trigger a destroy sequence of the meeting itself?

Reply:
my assumption was that the full locus sync will not fail, but you're right. I'll try to reproduce this scenario now that I understand better how to trigger the 409 error and confirm what happens

Reply:
added handling for the case if full locus also fails